production/user_guide.md

```diff
@@ -193,7 +193,7 @@ Once the job is completed, you will receive a `HTTP 200 Complete` response, whic
 
 <img class="ug-img" src="/assets/img/swagger_walkthrough_10.svg" alt="Swagger: copy the file name: the part of the URL after the last '/'" />
 
-To retrieve your data, open the `GET /data/{jobId}/{filename}` endpoint. Copy the `jobId` into the `jobId` field, and the last string of the URL received in the previous step (highlighted in green and dashed lines above) into the `filename` field, then hit `Execute`.
+To retrieve your data, open the `GET /data/{jobId}/{filename}.ndjson` endpoint. Copy the `jobId` into the `jobId` field, and the last string of the URL received in the previous step (highlighted in green and dashed lines above) into the `filename` field, then hit `Execute`.
```

Comment:
Since the file extension is technically part of the filename, this update isn't clear to me.

Also, we have this:
<img width="271" alt="Screen Shot 2020-05-01 at 11 48 13 AM" src="https://user-images.githubusercontent.com/37818548/80818763-c8052f80-8ba1-11ea-83d1-ff3d25a9b433.png">